data-structures-linked-lists/includes.js

```diff
@@ -0,0 +1,15 @@
+/* exported includes */
+
+function includes(list, value) {
+  while (list.next !== null) {
+    if (list.data === value) {
+      return true;
+    } else {
+      list = list.next;
+    }
+  }
+  if (list.data === value) {
+    return true;
+  }
```

Comment:
This shouldn't be needed here, the check in the `while` loop should catch everything